proto/frequenz/api/common/v1/microgrid/communication_components.proto

```diff
@@ -0,0 +1,71 @@
+// Frequenz microgrid communication components definitions.
+//
+// Copyright:
+// Copyright 2023 Frequenz Energy-as-a-Service GmbH
+//
+// License:
+// MIT
+
+syntax = "proto3";
+
+package frequenz.api.common.v1.microgrid.communication_components;
+
+
+// CommunicationComponent represents a communication component within a
```

Comment:
Nit: I would recommend putting the name of the struct in backticks, so that it gets formatted accordingly in the auto-generated docs.